src/plugins/base-exporters/index.js

```diff
@@ -40,16 +16,82 @@ var BaseExporters = function (config) {
     config.registerExporter('png24', path.join(__dirname, 'PNG.js'));
     config.registerExporter('png32', path.join(__dirname, 'PNG.js'));
     config.registerExporter('png256', path.join(__dirname, 'PNG.js'));
-    config.on('parseopts', this.parseOpts);
     config.addJS('/src/plugins/base-exporters/front/export.js');
-};
 
-BaseExporters.prototype.parseOpts = function (e) {
-    this.commands.export.option('format', {
-        help: 'Format of the export',
-        metavar: 'FORMAT',
-        default: 'xml',
-        choices: Object.keys(this.exporters)
+    config.commands.export = config.opts.command('export <project>')
+        .description('Export a project.')
+        .option('--mapnik-version [version]',
+            'Optional mapnik reference version to be passed to Carto.',
+            config.defaultMapnikVersion())
+        .option('--keep-cache',
```

Comment:
That's something I was unsure of. They are not really global as they only apply to serve and export, but not to plugins. Or am I mistaken here?